experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -2436,13 +2573,17 @@ describe("Editing", () => {
 		const anchor = cursor.buildAnchor();
 		cursor.free();
 
-		// Remove the root node
-		tree.editor.sequenceField(rootField).delete(0, 1);
-
 		// Fork the tree so we can undo the removal of the root without undoing later changes
+		// Note: if forking of the undo/redo stack is supported, this test can be simplified
```

Comment:
this test is still skipped as it fails for sequence field, but it was my understanding that we're likely to support the correct merge semantics here before we support forking of the undo/redo stack, and the same scenario was possible to model.

Reply:
Thanks, it's nice that the test doesn't need the undo/redo stack forking.
I don't know whether we'll want to supporting editing already removed content in this way anytime soon. It seems like a somewhat advanced use case.